examples/express-server/src/client.ts

```diff
@@ -1,5 +1,5 @@
 import { createTRPCClient, httpBatchLink, loggerLink } from '@trpc/client';
-import { tap } from '@trpc/client/observable';
+import { tap } from '@trpc/server/observable';
```

Comment:
It feels like we might want that `@trpc/core`-package to import from to prevent the feeling of importing from the server... need to think about this a bit. How do you feel?

Reply:
I agree but not sure about the DX for users having to install another package. I think it might be fine because it's only for those using subscriptions.

Reply:
A `@trpc/core`-package could also be a hard dependency (instead of peer) of `@trpc/server` + `@trpc/client` and just be a glue package for now.

Reply:
I think it's fine to leave this import as-is. I'm happy to open a future PR that pulls out any common logic/utility functions into a `@trpc/core` package if we decide this is best?